_includes/script.html

```diff
@@ -3,3 +3,18 @@
 <!-- Start of HubSpot Embed Code -->
 <script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/8551838.js"></script>
 <!-- End of HubSpot Embed Code -->
+
+<!-- This is installed from the https://github.com/slashdotdash/jekyll-lunr-js-search Gem -->
+<script src="{{ '/js/search.min.js' | relative_url }}" type="text/javascript" charset="utf-8"></script>
```

Comment:
Why `relative_url`? I'd have assumed it's always `/js/search.min.js`.

Reply:
ah yep, copy-paste error! ty :heart: